exonum/src/sandbox/sandbox_tests_helper.rs

```diff
@@ -207,7 +208,9 @@ pub fn compute_txs_merkle_root(txs: &[Hash]) -> Hash {
     let mut fork = MemoryDB::new().fork();
     let mut hashes = ProofListIndex::new("name", &mut fork);
     hashes.extend(txs.iter().cloned());
-    hashes.merkle_root()
+    hashes.merkle_root();
```

Comment:
What's the point of this code?

Reply:
Debugging artifact, I forgot to remove it.